packages/schema/src/config/app.ts

```diff
@@ -28,13 +28,6 @@ export default defineUntypedSchema({
      * @type {boolean}
      */
     propsDestructure: false,
-
-    /**
-     * Vue Experimental: Enable macro `defineModel`
-     * @see [Vue RFC#503](https://github.com/vuejs/rfcs/discussions/503)
-     * @type {boolean}
```

Comment:
As a note: Also happy to *deprecate* it to avoid breaking type-changes (as it does nothing anyway right now)